packages/website/src/components/editor/useSandboxServices.ts

```diff
@@ -71,6 +72,13 @@ export const useSandboxServices = (
           colorMode === 'dark' ? 'vs-dark' : 'vs-light',
         );
 
+        // registerInlayHintsProvider may not be present for older TS versions
```

Comment:
Yeah "inlay hints" were added in TS 4.4: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#inlay-hints

This strategy makes sense - if TS doesn't support them, we just won't enable them.

One request, to make this easier to find when we grep for version things to drop upon losing TS 4.3 support:

```suggestion
        // registerInlayHintsProvider was added in TS 4.4 and isn't in TS <= 4.3.
```

Reply:
Good point! Thanks